video_xblock/backends/base.py

```diff
@@ -224,7 +224,8 @@ def get_frag(self, **context):
         js_files += [
             'static/js/videojs/videojs-tabindex.js',
             'static/js/videojs/toggle-button.js',
-            'static/js/videojs/videojs-event-plugin.js'
+            'static/js/videojs/videojs-event-plugin.js',
+            'static/js/videojs/fullscreen-extends.js'
```

Comment:
Could you please add a trailing comma to the last list element?

Reply:
have been added